src/tokens/Assets.tsx

```diff
@@ -266,8 +267,8 @@ AssetsScreen.navigationOptions = {
 const styles = StyleSheet.create({
   listHeaderContainer: {
     ...getShadowStyle(Shadow.SoftLight),
-    paddingHorizontal: Spacing.Thick24,
-    paddingTop: Spacing.Smallest8,
+    paddingHorizontal: Spacing.Regular16,
```

Comment:
should we keep these at 24 for now, since the token balance items are spaced at 24?

Reply:
I think the header being different from the other tabs is more noticeable the the spacing below. We could do this conditionally for the bottom tab.

Reply:
Made it conditional base on the new tab view in 54e3825fe5a00cc63d3ddd37096625fe6f3e18f7. The difference in header placement was a bit to noticeable in my opinion. Once we have a follow up PR to adjust padding we can remove the non tab navigator path.